Tests/RxSwiftTests/PrimitiveSequence+DebugTest.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  PrimitiveSequence+DebugTest.swift
+//  Rx
+//
+//  Created by muukii on 3/13/17.
+//  Copyright © 2017 Krunoslav Zaher. All rights reserved.
+//
+
+import XCTest
+import RxSwift
+import RxTest
+
+class PrimitiveSequenceDebugTest : RxTest {
+
+}
+
+// MARK: debug
+extension PrimitiveSequenceDebugTest {
+  func testDebug_producesSingleElement() {
+    let singleResult: Single<Int> = Single.just(1)
+      .debug()
+
+    let result = try! singleResult
+      .toBlocking().first()!
+    XCTAssertEqual(result, 1)
+  }
+
+  #if TRACE_RESOURCES
+  func testReplayNReleasesResourcesOnComplete() {
```

Comment:
I think these unit tests have wrong names. I don't think we even need to add them in this case since we are only reusing implementation, but if you want to add them, please rename them.